### DISCUSSION TITLE
Request Names in Flow lost after restart

### DISCUSSION THREAD

**Discussion by Niecke** [postmanlabs/postman-flows#177] [Defect]
After restarting the Postman on Win10 the names of my Requests in my Flows are gone. But it looks like the Requests are still working (see Screenshot). I have recently updated to 9.6.

<img width="352" alt="image" src="https://user-images.githubusercontent.com/37218693/146829931-6b115dc9-3709-435d-a3c2-c638d087f23c.png">

**Comment by saswatds** [comment]
@Niecke thanks for reporting this. We are taking a look at this ASAP. Could you help me with any error logs from your devtools?

**Reply by Niecke** [reply]
@saswatds Thank you very much for the quick answer. Are the renderer-requester.logs sufficient or do you also need other logs?
I have also looked at the flows via the Web UI and the names are missing there as well.

[renderer-requester.log](https://github.com/postmanlabs/postman-flows/files/7756625/renderer-requester.log)

**Reply by saswatds** [reply]
Hey @Niecke, we were able to find the issue here. This is one of those bugs that always existed but showed when things got fixed.
* This bug will not affect any new blocks that are created.
* We are going to try and fix this from our backend, but in the meanwhile, you could just re-select the request and it should fix it.
* Request execution is not affected by this issue.

**Reply by Niecke** [reply]
Just had a quick look at a new flow and the bug seems to be gone. I will just re-select the requests in the old flows. Thanks for the really quick fix.

**Comment by ricky7171** [comment]
Hi. I also got same bug. Please fix this asap
